### ISSUE TITLE
PDS viewer and Arrays

### ISSUE THREAD

**Issue by dcoia740** [Small-Bodies-Node/pds4_tools#33]
PDS Class Array allows for names of array axes using attribute axis_name. 

Although axis_name is present in the label, it gets "ignored" by the PDS viewer when displaying the array by clicking on the Table bottom. (current button available: Label, Table, Plot)

It would be good if the axis name/s were recognised by the viewer also when displaying the array (and not just in the label)

Thank you
Daniela

**Comment by LevN0** [comment]
In the table view (for an array), you would the axis name as the row/column label? Or where would the axis name get used?

**Comment by dcoia740** [comment]
yes please

**Comment by dcoia740** [comment]
Also should there not be an "Array" button? Current available ones are Label, Table, Plot

**Comment by LevN0** [comment]
The buttons available suggest your data is a 1D array. They allow the following: 

Label - View the label for the product
Table - View the tabular form for the array
Plot   - Plot the 1D array against "row" number

What would an "Array" button do?

**Comment by dcoia740** [comment]
yes ok was just wondering, taking Table as its PDS type... never mind that part
